packages/unchained-client/src/evm/bnbsmartchain/parser/bep20.ts

```diff
@@ -58,9 +57,9 @@ export class Parser<T extends Tx> implements SubParser<T> {
 
     switch (txSigHash) {
       case this.supportedFunctions.approveSigHash: {
-        const amount = decoded.args.amount as BigNumber
+        const amount = decoded.args.amount as BigInt
         const value = amount.toString()
-        if (amount.isZero()) {
+        if (amount === BigInt(0)) {
```

Comment:
likewise here, can use 0n if you like tiny text

Reply:
48323359c2